test-data/unit/check-attr.test

```diff
@@ -1049,3 +1049,18 @@ class B:  # E: Function is missing a type annotation for one or more arguments
 
 reveal_type(B)  # N: Revealed type is 'def (x: Any) -> __main__.B'
 [builtins fixtures/list.pyi]
+
```

Comment:
Can you check that there are tests that run multiple semantic passes with various kinds of decorators? There is a small risk that this might break something.

Reply:
I run all test cases where top-levels processed twice (second time gives an extra iteration). I have found no failures related to this change. I however added another `attrs` specific decorator test case just in case.